app/mesh/changelog.md

```diff
@@ -1,608 +1,6 @@
 ---
-title: Kong Mesh Changelog
+title: Release notes
```

Comment:
@lahabana Can we leave "Kong Mesh" as part of the title? We use it as the page title and in some meta tags

Reply:
done!